.github/workflows/ci_bundle.yml

```diff
@@ -0,0 +1,43 @@
+name: CI_package
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  lux-package:
+    runs-on: ubuntu-latest
+    steps:
+      - name: create release
+        id: create_release
+        uses: octokit/request-action@v2.x
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          route: POST /repos/{owner}/{repo}/releases
+          # headers:
+          owner: 'Geoportail-Luxembourg'
+          repo: 'luxembourg-geoportail'
+          tag_name: ${{ github.ref }}
+          # target_committish: ${{ github.ref }}
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
```

Comment:
version 16?

Reply:
well this was copied from the GH-pages script, what version should we use ? does it matter for building the bundle ?